examples/apps/cli/main.c

```diff
@@ -97,3 +98,22 @@ int main(int argc, char *argv[])
 
     return 0;
 }
+
+/*
+ * Provide, if required an "otPlatLog()" function
+ */
+#if ((OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_NONE) ||  (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_APP_STREAM))
```

Comment:
Is this correct? I guess we should not define `otPlatLog` function if output is set to `OPENTHREAD_CONFIG_LOG_OUTPUT_NONE`  (`none` means platform should provide the method, right?)...